modules/services/espanso.nix

```diff
@@ -25,41 +21,58 @@ in {
         defaultText = literalExpression "pkgs.espanso";
       };
 
-      settings = mkOption {
+      configs = mkOption {
         type = yaml.type;
-        default = { matches = [ ]; };
+        default = { default = { }; };
```

Comment:
Do we need a default?

Reply:
I am following previous config here. I guess there is no harm to set an empty default when the users do not provide anything.

Reply:
Oh, sorry, I was unclear. I meant: do we need a `default` that's not just `{}`? I.e., why the `default` inside the `default`?

Reply:
I use `default = { }; inside default is to create an empty `~/.config/espanso/config/default.yml` when it is not set.

Without it, Espanso will throw below error.

```
[ERROR] unable to load config

Caused by:
    missing config directory
```